src/week4/WordBreak.java

```diff
@@ -0,0 +1,22 @@
+package week4;
+
+public class WordBreak {
+
+	public static void main(String[] args) {
+		String[] dictionary = { "pear","salmon","foot","prints"};
+		String input = "salmonloop";
+		System.out.println(wordbreakcheck(dictionary, input));
+	}
+
+	private static boolean wordbreakcheck(String[] dictionary, String input) {
+		if(input.equals("")||(input==null))

```

Comment:
Hey devin!! Thank you for the valuable feedback. Will keep track of it for future programs.